engine/src/main/resources/application.conf

```diff
@@ -139,7 +139,39 @@ backend {
           }
         }
       }
-      //    },
+    }
+//    ,
+//    HtCondor {
```

Comment:
For some reason the tests fail complaining that `ClassNotFound`if any of the other backends are uncommented. I have not gone into the details of that yet.